sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuExec.scala

```diff
@@ -260,7 +262,7 @@ trait GpuExec extends SparkPlan with Arm {
   lazy val allMetrics: Map[String, GpuMetric] = Map(
     NUM_OUTPUT_ROWS -> createMetric(outputRowsLevel, DESCRIPTION_NUM_OUTPUT_ROWS),
     NUM_OUTPUT_BATCHES -> createMetric(outputBatchesLevel, DESCRIPTION_NUM_OUTPUT_BATCHES)) ++
-      additionalMetrics
+      udfRunTimeMetrics ++ additionalMetrics
```

Comment:
Do we need to worry about any execs that have overidden `allMetrics`?

Reply:
I think it should be the developer's job to take care of these if he want to override this `allMetrics`.